octobot_trading/enums.py

```diff
@@ -61,7 +76,25 @@ class OrderStatus(enum.Enum):
     EXPIRED = "expired"
     REJECTED = "rejected"
     UNKNOWN = "unknown"
-
+    ORDER_CANCELED = "ORDER_CANCELED"
+    ORDER_FILLED = "ORDER_FILLED"
+    PARTIALLY_FILLED_CANCELED = "PARTIALLY_FILLED_CANCELED"
+    ORDER_NEW = "ORDER_NEW"
+
+class CryptoFeedOrderStatus(enum.Enum):
+    # copied from cryptofeed.defines
```

Comment:
Why do we need a copy of a cryptofeed enum?

Reply:
As we use it in the parser, I wanted to make sure It'll never change without our knowledge

Reply:
ah yeah good idea, I'll do that